libs/application-generic/src/commands/base.command.ts

```diff
@@ -24,7 +25,7 @@ export abstract class BaseCommand {
         data: mappedErrors,
       });
 
-      throw new BadRequestException(mappedErrors);
+      throw new CommandValidationException(mappedErrors);
```

Comment:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
⚠ **issue:** Same issue as [here](https://github.com/novuhq/novu/pull/6648#discussion_r1808389206) regarding stringification of the errors‏